message PushMessageAck
{
    required string ids = 1;
}
